frontend/src/components/BottomSignUp/BottomSignUp.js

```diff
@@ -1,10 +1,16 @@
 import React, {Component } from 'react';
 import './BottomSignUp.css'
 import signupImage from "../../assets/soup-kitchen.jpg";
+import { withRouter } from 'react-router'
 
 class BottomSignUp extends Component {
 
+  
+
```

Comment:
You can remove these two lines. Probably all the whitespace added on this card is unnecessary.